x/evm/module.go

```diff
@@ -245,7 +245,7 @@
 			ctx.Logger().Error("failed to send wei surplus of %s to EVM module account", surplusWei)
 		}
 	}
-	am.keeper.SetTxHashesOnHeight(ctx, ctx.BlockHeight(), utils.Map(evmTxDeferredInfoList, func(i keeper.EvmTxDeferredInfo) common.Hash { return i.TxHash }))
+	am.keeper.SetTxHashesOnHeight(ctx, ctx.BlockHeight(), utils.Filter(utils.Map(evmTxDeferredInfoList, func(i keeper.EvmTxDeferredInfo) common.Hash { return i.TxHash }), func(h common.Hash) bool { return h.Cmp(ethtypes.EmptyTxsHash) != 0 }))
```

Comment:
how problematic is this warning?

Reply:
probably better to avoid the panic at this point - we didn't want things to fail silently during testing phase but now since we are approaching the launch it's better to remove anything that could cause an unrecoverable halt

Reply:
updated